src/spikeinterface/extractors/nwbextractors.py

```diff
@@ -377,16 +601,14 @@ def __init__(
         if stream_mode == "fsspec" and stream_cache_path is not None:
             stream_cache_path = str(Path(self.stream_cache_path).absolute())
 
-        self._electrical_series = electrical_series
-
         # set serializability bools
         if file is not None:
             # not json serializable if file arg is provided
             self._serializability["json"] = False
 
         self._kwargs = {
             "file_path": file_path,
-            "electrical_series_name": self._electrical_series_name,
+            "electrical_series_path": self.electrical_series_path,
```

Comment:
This is super annoying because we will no be able to open nwb recording dump to json from actual relase in next relase.
Could you make some manual tests (automatic test is too complicated)  on cross version compatibility ?

Reply:
Forget it. The kwargs is left in the signautre.